src/net/mod.rs

```diff
@@ -1,3 +1,6 @@
+//! Net module holds all sockets
+
+pub mod udp;
```

Comment:
There are two `mod udp`.

Reply:
I think this happened when I pulled in the changes from your PR @TimonPost. Shockingly, I think it did compile.